packages/relay-provider/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { RelayResponse } from '@gelatonetwork/relay-sdk'
+import { BigNumber } from 'ethers'
+
+// TO-DO: Duplicated. Remove local type and import from "types" package
```

Comment:
Let's keep these types as they are now, knowing that in the future we should have one package where all the common files are exported. Right now I just prefer not adding them to the Core SDK types package as these may change a few times in the following weeks.